zapcore/facility.go

```diff
@@ -41,6 +41,19 @@ type Facility interface {
 	Write(Entry, []Field) error
 }
 
+type nopFacility struct{}
+
+// NopFacility returns a no-op Facility.
+func NopFacility() Facility { return nopFacility{} }
+
+func (nopFacility) Enabled(Level) bool { return false }
+
```

Comment:
Sounds good. I usually only do that when the methods have some structural similarity, but it's probably an improvement here too.